atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -364,6 +395,7 @@ def get_registry_session(self, registry):
 
     def run(self):
         digests = dict()
+        self.check_existing_vr_tag()
```

Comment:
The design related to this PR says this check should be performed in `tag_manifest_into_registry`, although it seems that with this approach we would avoid tagging something in a registry prior to raising an error (since `tag_manifest_into_registry` loops through registries) The downside is to connect to each registry here and back again below. Is this the reason you decided for this implementation?

Reply:
I just wanted to separate logic, and 1st do this check
and if all is good, we will proceed with tagging 

keep in mind that "looping through registries", there is more likely just 1 registry